From 279b4f57263abcdb84e2386f4cb4256981fb8c2d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Pavel=20B=C5=99ezina?= <pbrezina@redhat.com>
Date: Thu, 4 Aug 2016 14:10:09 +0200
Subject: [PATCH 31/39] dyndns: fix typo and unify ipa with ad debug message
 when off
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Reviewed-by: Petr Čech <pcech@redhat.com>
Reviewed-by: Lukáš Slebodník <lslebodn@redhat.com>
(cherry picked from commit b3851e86af91dc1aa6e265d5b2e4279b2611ff43)
---
 src/providers/ad/ad_dyndns.c | 2 +-
 src/providers/ipa/ipa_init.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/providers/ad/ad_dyndns.c b/src/providers/ad/ad_dyndns.c
index e3f1812837f7cee9d18ef001233871e0fcc16b4c..00190485e8f0ca7362ed60b2df022c74c53988c9 100644
--- a/src/providers/ad/ad_dyndns.c
+++ b/src/providers/ad/ad_dyndns.c
@@ -47,7 +47,7 @@ errno_t ad_dyndns_init(struct be_ctx *be_ctx,
 
     if (dp_opt_get_bool(ad_opts->dyndns_ctx->opts,
                         DP_OPT_DYNDNS_UPDATE) == false) {
-        DEBUG(SSSDBG_CONF_SETTINGS, "Dynamic DNS updates not set\n");
+        DEBUG(SSSDBG_CONF_SETTINGS, "Dynamic DNS updates are off.\n");
         return EOK;
     }
 
diff --git a/src/providers/ipa/ipa_init.c b/src/providers/ipa/ipa_init.c
index ca99200a1ba1d9508ac0affffecaa08149552fee..7dec4d1fb8541a48470d4e44f10838e5bea67ad5 100644
--- a/src/providers/ipa/ipa_init.c
+++ b/src/providers/ipa/ipa_init.c
@@ -200,7 +200,7 @@ static errno_t ipa_init_dyndns(struct be_ctx *be_ctx,
     enabled = dp_opt_get_bool(ipa_options->dyndns_ctx->opts,
                               DP_OPT_DYNDNS_UPDATE);
     if (!enabled) {
-        DEBUG(SSSDBG_CONF_SETTINGS, "Dynamic DNS updates are of.\n");
+        DEBUG(SSSDBG_CONF_SETTINGS, "Dynamic DNS updates are off.\n");
         return EOK;
     }
 
-- 
2.9.3

